test/deploy_and_test.sh

```diff
@@ -29,7 +29,7 @@ create_splunk_secret() {
 create_splunk_indexes() {
   splunk_ip=$1
   splunk_password=$2
-  index_names=("em_metrics" "em_meta" "em_logs")
+  index_names=("em_metrics" "em_meta" "em_events")
```

Comment:
With change of the index poller should deal on its own, right? it will see that ocnf is changed and update jobs, correct?

Reply:
During installation of SC4SNMP indexes are not created. This script is solely for purpose of our tests. This was the last part that was skipped during refactoring.

The install.bash script accepts:
    METRICS_INDEX=em_metrics \
    EVENTS_INDEX=em_events \
    META_INDEX=em_meta \
which are later used by our software.